kubemarine/admission.py

```diff
@@ -592,14 +592,14 @@ def manage_pss_enrichment(inventory, cluster):
     if "exemptions" in procedure_config:
         default_merger.merge(inventory["rbac"]["pss"]["exemptions"], procedure_config["exemptions"])
     if "namespaces" in procedure_config:
-        for namespace in procedure_config["namespaces"]:
+        for namespace_item in procedure_config["namespaces"]:
             # check if the namespace has its own profiles
-            if isinstance(namespace, dict):
-                profiles = list(namespace.values())[0]
-                for item in profiles:
-                    if item.endswith("version"):
-                        verify_version(item, profiles[item], minor_version)
-                raise Exception("Custom labels for each namespace are currently not supported")
+            if isinstance(namespace_item, dict):
+                namespace = list(namespace_item.keys())[0]
+                for item in list(namespace_item[namespace]):
+                    if namespace_item[namespace][item]:
+                        if item.endswith("version"):
+                            verify_version(item, namespace_item[namespace][item], minor_version)
```

Comment:
The only necessary change here is to remove raise Exception("..."). All other changes seem not necessary, but I personally find the code a bit more complicated.

Reply:
Actually, the following YAML has a significant difference:
```yaml
  namespaces: 
    - example-namespace-2:
        enforce: baseline
```
```yaml
  namespaces: 
    - example-namespace-2:
      enforce: baseline
```

Reply:
The code might be simpler if the item of namespaces list is restricted by a dictionary type. Instead the following YAML:
```yaml
  namespaces: 
    - example-namespace-1
    - example-namespace-2:
        enforce: baseline
```
it would be:
```yaml
  namespaces: 
    - example-namespace-1: {}
    - example-namespace-2:
        enforce: baseline
```
and we could exclude several condition checks